lib/mortar/command.rb

```diff
@@ -102,23 +104,21 @@ def variables_struct
     def dotted_path_to_hash(hash)
       hash.map do |pkey, pvalue|
         pkey.to_s.split(".").reverse.inject(pvalue) do |value, key|
-          {key.to_sym => value}
+          { key.to_sym => value }
         end
       end.inject(&:deep_merge)
     end
 
     # @return [K8s::Client]
     def client
-      return @client if @client
```

Comment:
@kke why did we remove this?

Reply:
It was prettier than the type of syntax rubocop would have wanted:

```
@client = if ENV['xyz']
                    foofoo
                 elsif bar?
                    xyz
                 end
```

and the memoization was not used for anything.